pkg/kwokctl/k8s/feature_gates_data.sh

```diff
@@ -16,20 +16,32 @@
 # https://github.com/kubernetes/website/blob/main/content/en/docs/reference/command-line-tools-reference/feature-gates.md
 # Some details of the feature gate on the official website are not synchronized so here we use a script to get the real details in the code
 
-latest_release="${1:-24}"
+set -o errexit
+set -o nounset
+set -o pipefail
+
+DIR="$(dirname "${BASH_SOURCE[0]}")"
+
+latest_release="${1:-}"
+minimum_release=6
+
+if [[ ! "${latest_release}" -gt "${minimum_release}" ]]; then
+  echo "Must be greater than ${minimum_release}"
+  exit 1
+fi
 
 function features() {
-    for i in $(seq 6 "${latest_release}"); do
-        curl -sSL "https://github.com/kubernetes/kubernetes/raw/release-1.${i}/pkg/features/kube_features.go" |
-            grep "{Default: " |
-            sed -e 's/\w\+\.//g' |
-            sed -e 's/[:,}]//g' |
-            awk "{print \$1, \$5, $i}"
-    done
+  for i in $(seq "${minimum_release}" "${latest_release}"); do
+    curl -sSL "https://github.com/kubernetes/kubernetes/raw/release-1.${i}/pkg/features/kube_features.go" |
```

Comment:
nit: maybe we should consider not hard coding 1 here but not required.

the minimum_release to latest_release should include the 1.

Reply:
Kubernetes added Feature Gate in 1.6

Reply:
It's not easy to calculate Semver